tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -711,7 +712,10 @@ def test_show_queue_counters(self, setup, setup_config_mode, duthosts, enum_rand
                         if hostname != duthost.hostname:
                             continue
                     # The interface name is always the last but one field in the BUFFER_QUEUE entry key
-                    interfaces.add(fields[-2])
+                    t2_multi_asic_match = duthost.is_multi_asic and fields[-3] == asic.namespace and \
+                        tbinfo['topo']['type'] == 't2'
```

Comment:
@sanjair-git, will this check work for single or multi asic voq pizzaboxes used as upperspine?